screens/Menu/CourseSchedule.js

```diff
@@ -23,53 +24,127 @@ import PropTypes from "prop-types";
  * @param  {} navigation props.navigation is the name of the object from Navigator library
  */
 function CourseSchedule(props) {
+    const [accessToken, setAccessToken] = React.useState(null);
+    const [selectedCalendarId, setSelectedCalendarId] = React.useState(null);
+    const [calendarEventsList, setCalendarEventsList] = React.useState(null);
+    const [switchVal, setSwitchVal] = React.useState("false");
+
+    //TODO Show only events after current date
+    let currentDate = new Date();
+
+    const getCalendarId = async () => {
+        let calendarId = await AsyncStorage.getItem("calendarId");
+        setSelectedCalendarId(calendarId);
+    };
+
+    const getAccessToken = async () => {
+        let accessToken = await AsyncStorage.getItem("accessToken");
+        setAccessToken(accessToken);
+    };
 
-     /**
+    const getSwitchValue = async () => {
+        let switchValue = await AsyncStorage.getItem('switchVal');
+        setSwitchVal(switchValue);
+    };
+
+    const getCalendarEvents = async () => {
+        let calendarEvents = await fetch('https://www.googleapis.com/calendar/v3/calendars/' + selectedCalendarId + '/events', {
+            headers: { Authorization: `Bearer ${accessToken}` }, singleEvents: true, orderBy: 'startTime'
+        });
+        let resp = await calendarEvents.json();
+        setCalendarEventsList(getFilteredGoogleCalendarEvents(resp));
+    }
+
+    const getFilteredGoogleCalendarEvents = (resp) => {
+        var filteredList = resp.items.map(element => {
+            return { id: element.id, summary: element.summary, description: element.description, location: element.location };
+        });
+        return filteredList;
+    };
+
+    const CourseScheduleScreen = true;
```

Comment:
what does this do?

Reply:
Its a parameter for the navigate method just bellow it. Chris used this convention in other components so i decided to follow it.